cms/api.py

```diff
@@ -59,7 +57,7 @@ def _verify_apphook(apphook, namespace):
         apphook_name = apphook.__class__.__name__
     elif hasattr(apphook, '__module__') and issubclass(apphook, CMSApp):
         return apphook.__name__
-    elif isinstance(apphook, string_types):
+    elif isinstance(apphook, str):
```

Comment:
I'm not sure if this is ok for Strings, I know you can't do this with list because Strings can be True when checking against a type of List.